03-intro-to-python.Rmd

```diff
@@ -1,9 +1,298 @@
 
 # Intro to Python
 
+Python is a popular programming language that was created by Guido van Rossum and released in 1991. 
+
+Python is supported by multiple libraries that support data science tasks:
+
+- NumPy for numerical computing with multidimensional arrays
+- pandas for data manipulation and analysis with data frames
+- Matplotlib for data visualization
+
+## Main Differences between R and Python
+
+The main difference between Python and R is that Python is a general-purpose programming language, while R is a statistical programming language. This means that Python is good at multiple things and can do most things, whereas R is very good at statistical analysis, but not as good at other things as Python.
```

Comment:
```suggestion
The main difference between Python and R is that Python is a general-purpose programming language, while R is a statistical programming language. This means that Python is good at multiple things and can do most things, whereas R is very good at statistical analysis, but not as good at other things as Python.
```

`but not as good at other things as Python` should be made a bit more specific. This makes it sound like R is ONLY good at statistics and nothing else. And it can be used for other things. I would focus on the idea that Python is probably better for bigger datasets than R but R has a lot of "ready to go" types of analyses and has more easy to ouse options for data visualization.

Reply:
I don't know what "big data" tools there are in Python (maybe @tillahoffmann knows), but I know that in R, you can use [arrow](https://r4ds.hadley.nz/arrow) for processing big datasets.